[CORL-3202] don't show the `Refresh comments` button when live updates are disabled

nick-funk opened the pull request:
## What does this PR do?

- turns off the `Refresh comments` button when live updates are disabled in the tenant settings

## These changes will impact:

- [X] commenters
- [X] moderators
- [X] admins
- [ ] developers

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

N/A

## How do I test this PR?

- disable live updates under `Admin > Configure > Advanced > Comment stream live updates`
- go to stream
- leave stream tab and come back
- no `Reload comments` button/prompt should show

## Were any tests migrated to React Testing Library?

No

## How do we deploy this PR?

- Merge into `develop`

nick-funk committed:
don't show the `Refresh comments` button when live updates are disabled

nick-funk committed:
set test id for UserBanPopover's close button

kabeaty reviewed (commented):
Hmm when testing these changes, if the live setting is enabled again in the admin, I'm not seeing the refresh comments button showing then as expected. Are you able to reproduce that?

nick-funk commented:
> Hmm when testing these changes, if the live setting is enabled again in the admin, I'm not seeing the refresh comments button showing then as expected. Are you able to reproduce that?

I am unfortunately not. I did the following to test:

- started up a fresh tenant
- went to stream and signed in
- posted a comment
- switched tabs
- came back
- stream was showing `Refresh comments`
- opened a new tab
- went to `admin > config > advanced > Comment stream live updates`
- disabled the live updates feature
- saved changes
- go back to stream tab
- hard refreshed my stream
- switched tabs
- came back
- I no longer see the `Refresh comments` button
- again, went to `admin > config > advanced > Comment stream live updates`
- enabled the live updates feature
- saved changes
- go back to stream tab
- hard refreshed my stream
- switched tabs
- came back
- I once again see the `Refresh comments` button in stream

kabeaty reviewed (approved):
Got it working, looks good!
